config/webpack.proxy.config.js

```diff
@@ -0,0 +1,37 @@
+const config = require('@redhat-cloud-services/frontend-components-config');
+const { setHtmlPlugin, setReplacePlugin, setCommonPlugins } = require('./build.plugins');
+const { setupDotenvFilesForEnv } = require('./build.dotenv');
+const setProxyRoutes = require('./spandx.config');
+
+const { _BUILD_RELATIVE_DIRNAME, DEV_BRANCH, DEV_PORT } = setupDotenvFilesForEnv({
+  env: 'proxy'
+});
+
+let BETA_PREFIX = '';
+
+if (/(prod|qa|ci)-beta/.test(DEV_BRANCH)) {
+  BETA_PREFIX = '/beta';
+}
+
+const { config: webpackConfig, plugins } = config({
+  appUrl: [`${BETA_PREFIX}/insights/subscriptions`, `${BETA_PREFIX}/openshift/subscriptions`],
+  debug: true,
+  deployment: (/beta/.test(BETA_PREFIX) && 'beta/apps') || 'apps',
+  env: (/(prod|qa|ci)(-stable|-beta)$/.test(DEV_BRANCH) && DEV_BRANCH) || 'ci-stable',
+  port: Number.parseInt(DEV_PORT, 10),
+  rootFolder: _BUILD_RELATIVE_DIRNAME,
+  routes: setProxyRoutes({ DEV_PORT, BETA_PREFIX }),
+  skipChrome2: true,
```

Comment:
@Hyperkid123 another question about `skipChrome2` having to be set in order get everything running?